gateleen-core/src/main/java/org/swisspush/gateleen/core/http/LocalHttpServerResponse.java

```diff
@@ -130,6 +136,10 @@ public HttpClientResponse exceptionHandler(Handler<Throwable> handler) {
 
     public LocalHttpServerResponse(Vertx vertx, Handler<HttpClientResponse> responseHandler) {
         super(vertx);
+        // Attach most simple possible exception handler to base.
+        setExceptionHandler(thr -> {
```

Comment:
`thr` is a very uncommon name here. Consider to use `ex` or `e` - even when it's Thowable

Reply:
Would name `throwable` (fully written) also be an option? I would prefer that because:

1. By fact an `Exception is a  Throwable` but a `Throwable is NOT an Exception`. Therefore name `e`  would be a lie.

2. This would feel like:

```
interface Pet {}
class Dog implements Pet {}
class Cat implements Pet {}

class EquivalentOfOurErrorHandler {
    void thisIsLikeOurErrorhandler(Pet cat){
        // Takes a 'Pet' but names the variable 'cat' -> what?
        // Hmm Essentially it would be the same if we rename 'thr' to 'ex' in our case.
        // IMHO this sounds somewhat silly. Doesn't it? :)
    }
}
```

Reply:
Ok then, you're right. Let's keep `thr` - this is not a show-stopper